lib/cli/watch-run.js

```diff
@@ -167,8 +168,36 @@ const createWatcher = (
   // we handle global fixtures manually
   mocha.enableGlobalSetup(false).enableGlobalTeardown(false);
 
-  const watcher = chokidar.watch(watchFiles, {
-    ignored: watchIgnore,
+  const watchFilesFlat = [];
```

Comment:
[Performance] This should be a `Set` so that lookups are an O(1) `.has()`, not an O(N) `.includes`. The `watchFilesFlat.includes(filePath)`s under `watcher.on('all', ...)` strike me as a likely hot path for repositories with many files.

Reply:
I learnt something. Thanks.